apps/silverback-website/docs/gatsby/formik.mdx

```diff
@@ -0,0 +1,99 @@
+# Formik
+
+[Formik](https://formik.org/) is the React form handling framework choice for
+Silverback. We chose Formik for how you can handle normal and complex form with a nice structure.
+
+## Quick intro
+
+### Declarative
+Formik takes care of the repetitive and annoying stuff—keeping track of values/errors/visited fields, orchestrating validation, and handling submission—so you don't have to. This means you spend less time wiring up state and change handlers and more time focusing on the project.
+
+### Intuitive
+No fancy subscriptions or observables under the hood, just plain React state and props. By staying within the core React framework and away from magic, Formik makes debugging, testing, and reasoning about your forms a breeze. If you know React, and you know a bit about forms, you know Formik!
+
+### Adoptable
+Since form state is inherently local and ephemeral, Formik does not use external state management libraries like Redux or MobX. This also makes Formik easy to adopt incrementally and keeps bundle size to a minimum.
+
+[Basic Formik form example](https://codesandbox.io/s/dazzling-swanson-wne32?from-embed)
+
+## Fields validation
+
+for fields validation we chose [Yup](https://github.com/jquense/yup), is a JavaScript schema builder for value parsing and validation. Define a schema, transform a value to match, validate the shape of an existing value, or both. Yup schema are extremely expressive and allow modeling complex, interdependent validations, or value transformations.
+
+[Basic Formik and Yup example](https://codesandbox.io/s/zkrk5yldz)
+
+## Custom fields
+To use your custom input field with formik you need to use the Field prop `component`.
+
+Custom field (that handles label, input type and errors) example:
+
+```jsx
+import React from "react"
+
+export function FormikText({
+  field, // { name, value, onChange, onBlur }
+  form: { touched, errors }, // also values, setXXXX, handleXXXX, dirty, isValid, status, etc.
+  ...props
+}) {
```

Comment:
Since we almost use arrow functions in silverback, I think it's better to also have examples which use them:

```suggestion
export const FormikText = ({
  field, // { name, value, onChange, onBlur }
  form: { touched, errors }, // also values, setXXXX, handleXXXX, dirty, isValid, status, etc.
  ...props
}) => {
```

Reply:
Updated 👍